_templates/resources.html

```diff
@@ -27,6 +27,9 @@ <h2 id="learn">{% translate learn %}</h2>
         <p>
           <a href="https://www.khanacademy.org/economics-finance-domain/core-finance/money-and-banking/bitcoin/v/bitcoin-what-is-it">Khan Academy</a>
         </p>
+        <p>
+          <a href="https://www.binance.vision">Binance Academy</a>
```

Comment:
Thank you for noting that there are now a few links to articles about Bitcoin, however, the homepage is still clearly more-so about getting started with blockchain technology and cryptocurrencies in general, rather than specifically about getting started with Bitcoin in an academic sense for the purposes of being on the Bitcoin.org Resources page:

<img width="1271" alt="image" src="https://user-images.githubusercontent.com/1130872/57144200-1675c780-6db0-11e9-8eb7-3aba140f540a.png">

Please remove.